COPYRIGHT

```diff
@@ -1,34 +1,1073 @@
-diceware -- Python implementation
-Copyright (C) 2015-2018 Uli Fouquet and contributors
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: diceware
+Upstream-Contact: Uli Fouquet
+Source: <https://github.com/ulif/diceware/>
 
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
+Files: *
+Copyright: 2015-2018, Uli Fouquet
+License: GPL-3+
 
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
+Files: diceware/wordlists/wordlist_en_securedrop.asc
+Copyright: 1995-2016 Arnold G. Reinhold <agr@mac.com>
+           2016-2017 Heartsucker <heartsucker@autistici.org>
+Source: https://github.com/heartsucker/diceware/blob/develop/wordlists/en_US/wordlist.txt
+License: Expat
 
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>.
+files: diceware/wordlists/wordlist_en.txt
```

Comment:
nitpicking, but this should be capitalized i believe.

Reply:
Fixed.